server/migrations/20210524210332-TestQueueChanges.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.renameTable('TestPlan', 'TestPlanVersion', {
+                transaction
+            });
+
+            await queryInterface.renameColumn(
+                'TestPlanVersion',
+                'publishStatus',
+                'status',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanVersion',
+                'sourceGitCommitHash',
+                'gitSha',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanVersion',
+                'sourceGitCommitMessage',
+                'gitMessage',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanVersion',
+                'createdAt',
+                'updatedAt',
+                { transaction }
+            );
+
+            await queryInterface.renameColumn(
+                'TestPlanReport',
+                'publishStatus',
+                'status',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanReport',
+                'testPlan',
+                'testPlanVersionId',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanReport',
+                'testPlanTarget',
+                'testPlanTargetId',
+                { transaction }
+            );
+            await queryInterface.removeColumn(
+                'TestPlanReport',
+                'coveragePercent',
+                { transaction }
+            );
+
+            await queryInterface.renameColumn(
+                'TestPlanRun',
+                'tester',
+                'testerUserId',
+                { transaction }
+            );
+            await queryInterface.renameColumn(
+                'TestPlanRun',
+                'testPlanReport',
+                'testPlanReportId',
+                { transaction }
+            );
+            await queryInterface.removeColumn(
+                'TestPlanRun',
+                'isManuallyTested',
+                { transaction }
+            );
+            await queryInterface.addColumn(
+                'TestPlanRun',
+                'results',
+                Sequelize.DataTypes.JSONB,
```

Comment:
May want to use a JSONB array here instead? To take advantage of the built in postgres handlers that will come with querying a collection of a json dataset.
Unless `results` will have some other shape beyond just being an array of test results to warrant that.

Reply:
Ah, good idea! I didn't know about that.